src/cli/backup/sharepoint.go

```diff
@@ -70,7 +70,11 @@ corso backup delete sharepoint --backup 1234abcd-12ab-cd34-56de-1234abcd`
 
 	sharePointServiceCommandDetailsExamples = `# Explore <site>'s files from backup 1234abcd-12ab-cd34-56de-1234abcd
 
-corso backup details sharepoint --backup 1234abcd-12ab-cd34-56de-1234abcd --site <site_id>`
+corso backup details sharepoint --backup 1234abcd-12ab-cd34-56de-1234abcd --site <site_id>
+# Explore site's files created before end of 2015 from a specific backup
```

Comment:
This is a bit confusing.  why is 2015 relevant?  What should the user do if they were created before 2015?  Do we have broad-scale supportability differences before and after this epoch that we need documented?

Reply:
The year is not relevant. The example is to show the user how to look up a file from before a certain date.

Reply:
Ah, fair.  I'd nix the date in that case.  Ex:
```suggestion
# Find all site files that were created before a certain date.
```